### PR TITLE
Fix italian language recognition for tntvillage provider

### PR DESCRIPTION
**Pull request by gborri**

### CODE REVIEW

**Comment by gborri** [comment]
@miigotu why do not merge?

**Comment by gborri** [comment]
is a small fix to correct the langue selection. Now is not working correctly. in all case.

**Comment by fernandog** [comment]
"Failed — The Travis CI build failed"

**Comment by gborri** [comment]
@fernandog sorry, I haven't realized there were a problmem. Now has been fixed.

**Comment by fernandog** [comment]
squash commits please

**Comment by gborri** [comment]
@fernandog i don't know how to do it. tried once but didn't work. could you please help me?

**Comment by fernandog** [comment]
@miigotu ok to merge now?

File: sickbeard/providers/tntvillage.py [hunk 1]
```diff
@@ -146,7 +151,7 @@ def _doLogin(self):
 
         login_params = {'UserName': self.username,
                         'PassWord': self.password,
-                        'CookieDate': 1,
+                        'CookieDate': 0,
```
Comment (miigotu) [inline sickbeard/providers/tntvillage.py]: Why this change?

Comment (gborri) [inline sickbeard/providers/tntvillage.py]: @miigotu becouse i have no chance to make cookie work on tntvillage. I don't know how to make it work. It also happen that when sickrage start the first connection to tntvillage fail. i think that is due to old session.

**Comment by miigotu** [comment]
@gborri 
```
git fetch upstream
git rebase -i upstream/develop
```
change the line `pick 7047772 Update tntvillage.py` to `fixup 7047772 Update tntvillage.py`
ctrl+x, enter
```
git push -f origin patch-tntvillage_subtitle_recognition
```

**Comment by miigotu** [comment]
As a side note, from looking at this provider, I don't like how it is set up. Qualities have no place being parsed or defined in a provider class.

**Comment by miigotu** [comment]
@gborri As soon as you squash this I will merge it.

**Comment by gborri** [comment]
@miigotu i have tried your suggestion to squash commit. the first time it drop the second commit. Then I made the change again and committed. Now, using the same command, it gave me an error.

**Comment by gborri** [comment]
@miigotu I have succesful squash it, could you please merge it?
